driver/tke_driver.go

```diff
@@ -851,3 +964,97 @@ func (d *Driver) GetK8SCapabilities(ctx context.Context, opts *types.DriverOptio
 	}
 	return capabilities, nil
 }
+
+func getWrapAddClusterInstancesRequest(state, newState *state) (*ccs.AddClusterInstancesRequest, error) {
```

Comment:
Should we call it `getWrapUpdateClusterInstancesRequest`? it is used when updating cluster.